src/main/java/au/com/addstar/truehardcore/TrueHardcore.java

```diff
@@ -109,7 +109,7 @@
           Material.PODZOL,
           Material.GRASS,
           Material.GRASS_BLOCK,
-          Material.GRASS_PATH,
+          Material.DIRT_PATH,
```

Comment:
:clap: Nice! I had forgotten about this change.. might be a problem in several plugins.